src/modules/StatusLEDModule.cpp

```diff
@@ -26,7 +26,11 @@ int StatusLEDModule::handleStatusUpdate(const meshtastic::Status *arg)
                 power_state = charged;
             }
         } else {
-            power_state = discharging;
+            if (powerStatus->getBatteryChargePercent() > 5) {
```

Comment:
FWIW, I looked up LI ion battery curves and 5% seems to be the start of the precipitous drop, so feels like the right number here.